nucypher/characters/control/specifications/fields/messagekit.py

```diff
@@ -38,7 +38,20 @@ def _deserialize(self, value, attr, data, **kwargs):
             raise InvalidInputData(f"Could not parse {self.name}: {e}")
 
     def _validate(self, value):
+
+
+        if not isinstance(value, bytes):
+            value = b64decode(value)
         try:
-            UmbralMessageKitClass.from_bytes(value)
+            metadata = UmbralMessageKitClass.splitter().get_metadata(value)
+            if not UmbralMessageKitClass.splitter().validate_checksum(value):
+                if metadata['checksum'] in BYTESTRING_REGISTRY:
+                    raise InvalidInputData(f"Input data seems to be the bytes for a {BYTESTRING_REGISTRY[metadata['checksum']].__name__} and not a MessageKit")
+                raise InvalidInputData(f"Could not validate supplied MessageKit bytes against known any supported bytestring formats")
+
+            if metadata['version'] > UmbralMessageKitClass.version:
```

Comment:
I'd go further and disallow for the moment any other version that it's not the current one.
```suggestion
            if metadata['version'] != UmbralMessageKitClass.version:
```